pyproject.toml

```diff
@@ -74,13 +74,14 @@ redis = { version = "*", optional = true }
 saq = { version = "^0.9.1", optional = true }
 sentry-sdk = { version = "*", optional = true }
 sqlalchemy = { version = "==2.0.0rc3", optional = true }
+greenlet = {version = "*", platform = "darwin", optional = true }
```

Comment:
Is this for this?

https://github.com/sqlalchemy/sqlalchemy/issues/7714

Reply:
It is, I didn't think to look for an open issue on this.

Reply:
I wonder if the issue is arm64 in general, or just M1?

Reply:
I believe this is going to be an M1/OSX only specific issue.  I'd bet that the x86 OSX laptops probably do not have this issue.  Is there a way to specify architecture as well?

Reply:
I don't know - we are considering moving our workloads onto https://aws.amazon.com/ec2/graviton/ - so I might just spin one up and see if it gets the same treatment as m1